WebApp/autoreduce_webapp/instrument/views/runs.py

```diff
@@ -70,7 +69,7 @@ def submit_runs(request, instrument=None):
         return context_dictionary
 
 
-# pylint:disable=too-many-return-statements,too-many-branches,too-many-statements
+# pylint:disable=too-many-return-statements,too-many-branches,too-many-statements,too-many-locals
```

Comment:
I've got a feeling pylint is trying to tell us something 🤔

Reply:
Agree, but I felt refactoring this large function was out of scope I will open an issue for it.

Reply:
Yep definitely not in the scope of this PR